common/history/characters/0_dvg_lotharingia.txt

```diff
@@ -0,0 +1,162 @@
+﻿CHARACTERS = {
+	c:LOT = {
+		
+		
+		create_character = {

```

Comment:
What do you mean it's not allowed? In-game it seems to work, but is there a way to specifically enable that? 

I put him as both ruler and National Gendarmerie leader to make it more difficult to diminish the interest group's clout.

Reply:
I think because the next rules get elected from the interest group. Now he is also the heir. The ruler can't be the leader of an IG at the same time.
Maybe need to make your heir the IG leader instead, or create a new strong IG character that is popular.
Having the ruler be a member of that IG is enough for the bonus.

Reply:
aight did a bit of both